test/e2e/kubectl/kubectl.go

```diff
@@ -746,6 +746,23 @@ metadata:
 		})
 	})
 
+	framework.KubeDescribe("Kubectl get componentstatuses", func() {
+		It("should get componentstatuses", func() {
+			By("validating the result")
+			// get component list
+			output := framework.RunKubectlOrDie("get", "componentstatuses", "--no-headers=true")
+			lines := strings.Split(strings.TrimRight(output, "\n"), "\n")
+			// validate result for each component
+			for _, line := range lines {
+				component := strings.Split(line, " ")
+				result := framework.RunKubectlOrDie("get", "componentstatuses", "--no-headers=true", component[0])
+				if !strings.HasPrefix(result, component[0]) {
+					framework.Failf("Could not get componentstatuses")
```

Comment:
Necessary to output both result and component[0] into log message for debugging.

Reply:
I added these results.